src/frontend/components/SøknadsSteg/Oppsummering/Oppsummering.tsx

```diff
@@ -288,6 +306,324 @@ const Oppsummering: React.FC = () => {
+                                søknadsvar={
+                                    barn[barnDataKeySpørsmål.andreForelderPensjonUtland].svar
+                                }
+                            />
+                            {barn[barnDataKeySpørsmål.andreForelderPensjonHvilketLand].svar && (
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst
+                                            id={'ombarnet.andre-forelder.utenlandspensjon.land.spm'}
+                                            values={{ navn: barn.navn }}
+                                        />
+                                    }
+                                    søknadsvar={landkodeTilSpråk(
+                                        barn[barnDataKeySpørsmål.andreForelderPensjonHvilketLand]
+                                            .svar,
+                                        intl.defaultLocale
+                                    )}
+                                />
+                            )}
+                        </StyledOppsummeringsFeltGruppe>
+                        <StyledOppsummeringsFeltGruppe>
+                            <OppsummeringFelt tittel={<SpråkTekst id={'ombarnet.bosted'} />} />
+
+                            <OppsummeringFelt
+                                tittel={
+                                    <SpråkTekst
+                                        id={'ombarnet.bor-fast.spm'}
+                                        values={{ navn: barn.navn }}
+                                    />
+                                }
+                                søknadsvar={barn[barnDataKeySpørsmål.borFastMedSøker].svar}
+                            />
+                            <OppsummeringFelt
+                                tittel={
+                                    <SpråkTekst
+                                        id={'ombarnet.delt-bosted.spm'}
+                                        values={{ navn: barn.navn }}
+                                    />
+                                }
+                                søknadsvar={
+                                    barn[barnDataKeySpørsmål.skriftligAvtaleOmDeltBosted].svar
+                                }
+                            />
+                        </StyledOppsummeringsFeltGruppe>
+                        <StyledOppsummeringsFeltGruppe>
+                            <OppsummeringFelt
+                                tittel={
+                                    <SpråkTekst
+                                        id={'ombarnet.søker-for-periode.spm'}
+                                        values={{ navn: barn.navn }}
+                                    />
+                                }
+                            />
+
+                            {barn[barnDataKeySpørsmål.søkerForTidsromSluttdato].svar !==
+                            'UKJENT' ? (
```

Comment:
Vi har en constant for denne som kan brukes :)

Reply:
Nice, da legger jeg den til